test/src/main/java/org/apache/accumulo/test/ScanServerIT_NoServers.java

```diff
@@ -129,19 +123,10 @@ public void testScanOfflineTable() throws Exception {
       String tableName = getUniqueNames(1)[0];
 
       client.tableOperations().create(tableName);
-
```

Comment:
Same comment here as in ScanServerIT. You are changing the code path that's being tested.